gui/wxpython/psmap/dialogs.py

```diff
@@ -2512,26 +2504,21 @@ def _ColorsLinePanel(self, notebook):
         self.outlineCheck = wx.CheckBox(
```

Comment:
Please use the CheckBox from wrap.py here.

Reply:
Okay I used.